addons/sale_crm/models/crm_lead.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models, _
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    sale_amount_total= fields.Monetary(compute='_compute_sale_amount_total', string="Sum of Orders", currency_field='company_currency')
+    sale_number = fields.Integer(compute='_compute_sale_amount_total', string="Number of Quotations")
+    order_ids = fields.One2many('sale.order', 'opportunity_id', string='Orders')
+
+    @api.depends('order_ids')
+    def _compute_sale_amount_total(self):
+        for lead in self:
+            total = 0.0
+            nbr = 0
+            for order in lead.order_ids:
+                if order.state == 'draft':
+                    nbr += 1
+                if order.state not in ('draft', 'cancel'):
+                    total += order.currency_id.compute(order.amount_untaxed, self.company_currency)
+            lead.sale_amount_total = total
+            lead.sale_number = nbr
+
+    @api.model
+    def retrieve_sales_dashboard(self):
+        res = super(CrmLead, self).retrieve_sales_dashboard()
+        date_today = fields.Date.from_string(fields.Date.today())
```

Comment:
@dmo-odoo, There is no difference between `fields.Date.from_string(fields.Date.today())` and `date.today()` but  we prefere to use own odoo function.